.storybook/stories/useCameraShake.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import * as THREE from 'three'
+import { useFrame, Camera } from 'react-three-fiber'
+
+import { Setup } from '../Setup'
+
+import { useCameraShake, ShakeConfigPartial, PerspectiveCamera } from '../../src'
+
+export default {
+  title: 'Camera/useCameraShake',
+  component: useCameraShake,
+  decorators: [
+    (storyFn) => (
+      <Setup cameraPosition={new THREE.Vector3(0, 0, 10)} controls={false}>
+        {storyFn()}
+      </Setup>
+    ),
+  ],
+}
+
+function Scene() {
+  const cube = React.useRef<THREE.Mesh>()
+
+  useFrame(() => {
+    if (cube.current) {
+      cube.current.rotation.x = cube.current.rotation.y += 0.01
+    }
+  })
+
+  return (
+    <>
+      <mesh ref={cube}>
+        <boxBufferGeometry args={[2, 2, 2]} />
+        <meshStandardMaterial wireframe color="white" />
+      </mesh>
+      <mesh position={[0, -6, 0]} rotation={[Math.PI / -2, 0, 0]}>
+        <planeBufferGeometry args={[200, 200, 75, 75]} />
+        <meshBasicMaterial wireframe color="red" side={THREE.DoubleSide} />
+      </mesh>
+    </>
+  )
+}
+
+function UseCameraShakeDemo({ cfg }) {
+  const baseCamRef = React.useRef<Camera>()
+  const shakeCamRef = useCameraShake(baseCamRef.current, cfg)
+
+  useFrame(({ gl, scene }) => {
+    gl.render(scene, shakeCamRef.current)
```

Comment:
lets move this into the hook.

Reply:
@joshuaellis I made this change in the latest commit. I'm not sure it belongs inside the hook though. Since were giving it a render priority we are disabling automatic rendering. It feels like this hook shouldn't dictate the render priority. Just my take though, I'll leave it up to you.

Reply:
Hm. The hook doesn’t work without it being order 1 does it?

Reply:
Don't believe so, because it will render the scene out of the default camera (which isn't actually shaking) without the render priority.

Reply:
So maybe it's okay to highjack the render-order for this effect assuming there's an option to remove it's hierarchy? WDYT @gsimone

Reply:
we do this in `useCamera` I've decided it's a-okay lol.